src/protagonist/Portal/Pages/Queue/Upload.cshtml

```diff
@@ -0,0 +1,58 @@
+﻿@page
+@model Portal.Pages.Queue.IndexModel;
+
+@{
+    ViewData["Title"] = "Upload";
+}
+
+@section header
+{
+    <link href="https://cdn.jsdelivr.net/npm/dropzone@5.9.2/dist/min/dropzone.min.css" rel="stylesheet" />
+    <link href="~/assets/dist/css/jquery-ui.min.css" rel="stylesheet" />
+    <link href="~/assets/dist/css/jquery-ui.structure.min.css" rel="stylesheet" />
+}
+
+<div>On this page you can upload a CSV of information about images for the DLCS to ingest.</div>
+<a href="https://legacyportal.dlcs.digirati.io/Support/sample.csv">Click here to download a sample .csv file.</a>
```

Comment:
This shouldn't reference the old portal as it's a dev environment and will be removed once we roll out Protagonist.

Reply:
I overlooked this - I'll stick a copy of the file in wwwroot and link to there instead